docs/reference/migration/apis/deprecation.asciidoc

```diff
@@ -3,6 +3,11 @@
 [[migration-api-deprecation]]
 === Deprecation Info APIs
 
+IMPORTANT: Before performing a major version upgrade, make sure you have checked
+this API using the most recent minor version of the previous major version. In
```

Comment:
"the previous major version" <-- replace with "the major version you are upgrading from".